taskrunners/cleaner/Dockerfile

```diff
@@ -1,8 +1,6 @@
 FROM python:3.8.6
 WORKDIR /usr
 COPY requirements.txt requirements.txt
-COPY setup_requirements.sh setup_requirements.sh
-RUN ./setup_requirements.sh
```

Comment:
How are the requirements being install now?

Reply:
Oh wait, you're right. This is a different container than I thought. I'll find where that file went.

Reply:
Reverted the dockerfile and added the missing setup script.